charts/fybrik/values.yaml

```diff
@@ -296,14 +327,24 @@ opaServer:
   # the equivalent fields of .Values.global.podSecurityContext.
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
   podSecurityContext: {}
+  # for tls support add:
+  #  runAsNonRoot: false
+  #  runAsUser: 0
 
   # Container Security Context. If set, the fields of containerSecurityContext override
   # the equivalent fields of .Values.global.containerSecurityContext.
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
   containerSecurityContext: {}
+  # for tls support add:
+  # privileged: true
+  # allowPrivilegeEscalation: true
+  # capabilities:
+  #   add:
+  #     - NET_BIND_SERVICE
 
   service:
     type: ClusterIP
+    # For tls connection use port 443
```

Comment:
maybe 8443,

btw, why do we use 8181?

Reply:
443 is the port I saw was used in an example chart https://github.com/helm/charts/blob/master/stable/opa/values.yaml

Reply:
443 is the default port for https as 80 for http. With dockers we can use the default ports, but I saw that we useally use something like 8080

Reply:
ok i will try to change it to 8443.